src/components/Tooltip/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -644,51 +536,6 @@ exports[`Tooltip should match snapshot when visisble and direction is left 1`] =
 `;
 
 exports[`Tooltip should match snapshot when visisble and direction is right 1`] = `
-.c2.open-enter .c0.open-enter:before,
-.c2.open-appear,
-.c2.open-appear:before {
-  -webkit-transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  display: block;
-  opacity: 0;
-}
-
-.c3.open-enter .c0.open-enter:before,
-.c3.open-appear,
-.c3.open-appear:before {
-  -webkit-transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  display: block;
-  opacity: 0;
-}
-
-.c4.open-enter .c0.open-enter:before,
-.c4.open-appear,
-.c4.open-appear:before {
-  -webkit-transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  display: block;
-  opacity: 0;
-}
-
-.c5.open-enter .c0.open-enter:before,
-.c5.open-appear,
-.c5.open-appear:before {
-  -webkit-transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  display: block;
-  opacity: 0;
-}
-
-.c6.open-enter .c0.open-enter:before,
-.c6.open-appear,
-.c6.open-appear:before {
-  -webkit-transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  transition: opacity 0.25s cubic-bezier(0.25,0.46,0.45,0.94);
-  display: block;
-  opacity: 0;
-}
-
```

Comment:
Where did these styles go?

Reply:
It seems like removing of these styles is a fix for old ones. For some reason old styles were duplicated for every new instance.